pyproject.toml

```diff
@@ -8,6 +8,7 @@ show_missing = true
 [tool.pytest.ini_options]
 minversion = "6.0"
 log_cli_level = "INFO"
+filterwarnings = "ignore:::.*.traefik_k8s.*"
```

Comment:
Why this specific file? Shouldn't we ignore everything under `lib/`?

Reply:
I added this because I was getting flooded by traefik_k8s warnings from pydantic.

I'm not against widening this, but we would probably want to keep the warnings from our libraries?

I'm not super familiar with the syntax, so I'll need to figure out what we want to do here. I'll probably just remove this from the PR since I don't want it to hold anything up.

Reply:
https://github.com/canonical/vault-k8s-operator/pull/365/commits/017c5793763c874060d80a05410a0849d504ce2d